opentelemetry-sdk/src/opentelemetry/sdk/util/__init__.pyi

```diff
@@ -39,7 +39,8 @@ class BoundedList(Sequence[_T]):
     Calls to `append` and `extend` will drop the oldest elements if there is
     not enough room.
     """
-
+    dropped: int = ...
```

Comment:
Are the `...` needed here? I think this should work
```suggestion
    dropped: int
```

Reply:
Actually, I believe it needs to be removed. If mypy checked the stub directly, it would say `expression has type "ellipsis", variable has type "int"`

Reply:
Yes, this makes sense, checked it - and will update it